x-pack/plugins/cloud_security_posture/server/fleet_integration/fleet_integration.ts

```diff
@@ -50,31 +50,59 @@ export const onPackagePolicyPostCreateCallback = async (
   logger: Logger,
   packagePolicy: PackagePolicy,
   savedObjectsClient: SavedObjectsClientContract
+): Promise<void> => {
+  await UpdateCspRulesAccordingToCspRuleTemplates(packagePolicy, savedObjectsClient, logger);
+};
+
+export const OnPackagePolicyUpgradeCallback = async (
```

Comment:
This separation into two functions was intentional.

I do want to separate the events for both the reader and the developer to enhance them later.